tests/unittests/sources/test_openstack.py

```diff
@@ -737,20 +699,18 @@ def fake_dmi_read(dmi_key):
             assert False, "Unexpected dmi read of %s" % dmi_key
 
         m_dmi.side_effect = fake_dmi_read
-        ds = self._fake_ds()
+        ds = fake_ds
```

Comment:
Probably don't need to assign to another local var and just carry fake_ds to all asserts below

Reply:
Nevermind this, I realize now it's line wrap avoidance. Leaving it untouched.